nixos/modules/virtualisation/grow-partition.nix

```diff
@@ -1,6 +1,4 @@
-# This module automatically grows the root partition on virtual machines.
-# This allows an instance to be created with a bigger root filesystem
-# than provided by the machine image.
+# This profile is deprecated, use boot.growPartition directly.
```

Comment:
Perhaps replace this file with an entry in nixos/modules/rename.nix

Reply:
Good idea. Addressed in #33521